app/src/main/res/layout/item_timeline_conversation.xml

```diff
@@ -108,13 +112,13 @@
                 <TextView
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/dp_4"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 16) you should *also* add `android:layout_marginLeft="@dimen/dp_4"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.